src/App.js

```diff
@@ -10,16 +10,27 @@ class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies
+      movies: movieData.movies,
```

Comment:
I wouldn't generally recommend hard coding this in here - it's better to give it it's default value of an empty array, and then use setState to populate it. This will be especially true when you get the apiCalls working.